services/app-api/types/reports.ts

```diff
@@ -86,12 +86,26 @@ export enum MeasureTemplateName {
   "MLTSS-DM" = "MLTSS-DM",
   // pom measures
   "POM-1" = "POM-1",
+  "FFS-POM-1" = "FFS-POM-1",
```

Comment:
Should these be added to:
- `services/ui-src/src/types/report.ts`

As a result these may require the measures added to the following tests:
- `services/ui-src/src/utils/testing/setupJest.tsx`
- `services/ui-src/src/utils/state/reportLogic/reportActions.test.ts`
- `services/ui-src/src/components/report/ReportPageWrapper.test.tsx`